### ISSUE TITLE
Change URL

### ISSUE THREAD

**Issue by jarredmoore** [WWBN/AVideo#2742]
I want to change the url of my VOD instance. How can I do that??..

**Comment by akhilleusuggo** [comment]
what do you mean exactly ?

**Comment by DanielnetoDotCom** [comment]
I am also not sure what you mean.

do you mean change the domain of your site?

**Comment by jarredmoore** [comment]
Sorry so late..Yes change the domain name..

**Comment by akhilleusuggo** [comment]
Again , what do you mean ?
I understand you wanna change the domain name . So where is the problem ? What has your question to do with the script  ?

You need to change it by getting a new domain name from your provider ( DNS )
In case I'm understanding your question in a different way , you may wanna change ww.domain.com/AVideo to another thing like ww.domain.com/WhatEver
If this is the case , you need to setup a Vhost ( google it )

In case you have changed your domain name , but you're having problems .
`nano /var/www/html/AVideo/videos/configuration.php `
change the `$global['webSiteRootURL'] =` "yournewdomain"

**Comment by jarredmoore** [comment]
Sorry for not being clear. Yes I just wanted to change the global website url. I installed it using a test domian now i want to change it to a prodution domain. I'll try the command a bit later this afternoon.

**Comment by jarredmoore** [comment]
So I was able to change the domain name by editing the configuration.php file like suggested. I also configured my domian names to point to my digital ocean droplet, setup a few records (a,mx,cnames etc), changed/added a .conf config in sites available then enable site and restarted Apache and it works finda :)

I guess I was really asking about my Apache2 setup, dns, certbot setup and if I correctly pointed one or more domain names to the same folder on my server?? 

Simply Put:
I have ONE install of aVideo
/var/www/html/YouPHPTube

I want to point TWO domain names to ONE folder/instance.
www.vodone.com points to:
/var/www/html/YouPHPTube

www.vodtwo.com points to:
/var/www/html/YouPHPTube

Is this okay to do??

**Comment by akhilleusuggo** [comment]
Yes you can do that .
But this has nothing to do with this project , or at the least there's nothing that gonna really support you.
You need to google more about vhosts . Points them to the same directory . On the config :
```
ServerName www.vodone.com
Root /var/www/html/YouPHPTube
```

You can create a new one or add Alias in case you want to tell apache that www and non-www
`Alias www.vodtwo.com`

In case you wanna create a new config and put a new server name , go for it .
I'm not sure if the SSL gonna work fine , but this should work if you do it right .
